.golangci.yml

```diff
@@ -85,164 +62,142 @@ linters:
     - sloglint
     - sqlclosecheck
     - staticcheck
-    - stylecheck
     - tagalign
-    - tenv
     - testableexamples
     - testifylint
     - thelper
     - tparallel
-    - typecheck
     - unconvert
     - unparam
     - unused
     - usestdlibvars
     - wastedassign
     - whitespace
     - zerologlint
-    # - cyclop
-    # - depguard
-    # - exhaustruct
-    # - forbidigo
-    # - funlen
-    # - gochecknoglobals
-    # - gocognit
-    # - gomnd
-    # - gosec
-    # - lll
-    # - maintidx
-    # - nestif
-    # - nilnil
-    # - nlreturn
-    # - nonamedreturns
-    # - paralleltest
-    # - tagliatelle
-    # - testpackage
-    # - varnamelen
-    # - wrapcheck
-    # - wsl
```

Comment:
Let's keep this list as reference for future updates.

Reply:
But we dont plan to add/enable those linters right? I didnt understand why we should keep them :thinking:

Reply:
It's good for understanding "what's left". Also, when updating golangci-lint I diff the disabled linters to understand which ones are new and may be enabled without needing to fix anything.

Reply:
Got it. In that case I think will keep the default linters also commented so that we know whats there

Reply:
Done. Updated.